### DISCUSSION TITLE
Import config from another file?

### DISCUSSION THREAD

**Discussion by bschlenk** [starship/starship#3894] [Q&A]
Hey! Is there a way to import part of a config from another file? I use a separate config for work and for personal use, and thought it'd be cool to conditionally import extra configuration for my work setup. I didn't see anything in the docs (but didn't look very hard), so just wondering if this is possible.

**Comment by andytom** [comment]
Unfortunately not, at the moment starship only reads a single config file and can't import other files. There is an open issue for this but I can't seem to find it at the moment.
